### PR TITLE
exposing -I in sertop, fixes #53

### LINKED ISSUE #53: Why is `-I` not exposed by sertop?
Hey Emilio,

I'm trying to resurrect PeaCoq's code base, and am puzzled by how to replace the old `LibAdd`.

I can seem to get my plugin to start loading, by having `-Q /path/to/plugin,PluginName`, but unfortunately, my `plugin.ml4` opens a file in the same directory, that is compiled as a .cmxs, and Coq fails to know how to load it by default.

But, using CoqIDE, I can add `-I /path/to/plugin`, and then everything works well.

### PR DESCRIPTION
**Pull request by Ptival**
In case this is just the feature you want to fix, this works well for me.

### CODE REVIEW

File: sertop/sertop_arg.ml [hunk 1]
```diff
@@ -88,6 +88,15 @@ let load_path : Mltop.coq_path list Term.t =
...
 
+let coq_include_conv unix_path = Mltop.{
+    path_spec = MlPath unix_path;
+    recursive = false;
+  }
+
+let include_path : Mltop.coq_path list Term.t =
+  let doc = "Include DIR in default loadpath, for locating ML files" in
+  Term.(const List.(map coq_include_conv) $
+        Arg.(value & opt_all dir [] & info ["I"; "include-path"] ~docv:"DIR" ~doc))
```
Comment (ejgallego) [inline sertop/sertop_arg.ml]: Should we name this `ml-path` instead?

**Comment by ejgallego** [comment]
Thanks for the PR!

**Comment by Ptival** [comment]
Well, I'm not quite sure. Does Coq ever use the `-R/-Q` paths to resolve `.ml` files?

My setup has files like this:
```
`- peacoq-plugin
  `- PeaCoq.v
  `- peacoq_plugin.ml4
```

And in my `PeaCoq.v`, I only do `Declare Module "peacoq_plugin".`.

I wonder if I could somehow use the `PeaCoq` logical path. I will try and report my experience.

**Comment by ejgallego** [comment]
> I wonder if I could somehow use the PeaCoq logical path. I will try and report my experience.

Yup, it should, see:
```ocaml
let coq_lp_conv ~implicit (unix_path,lp) = Mltop.{
    path_spec = VoPath {
        coq_path  = Libnames.dirpath_of_string lp;
        unix_path;
        has_ml    = AddRecML;
        implicit;
      };
    recursive = true;
  }
```
So that indicates Coq to look for ml files recursively, so if `-Q/-R` don't add to the ML loadpath that would be a bug [which wouldn't surprise me due to the state of the corresponding Coq's code]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 _Version 0.5.2_:
 
  * [serlib] Compatibility with OCaml 4.07.0 [problems with `Stdlib` packing]
+ * [sertop] Support for `-I` option.
```
Comment (ejgallego) [inline CHANGELOG.md]: Please also add the long name `--ml-include-path` which should be auto-explanatory.

Comment (ejgallego) [reply CHANGELOG.md]: Version `0.5.2` has already been released, please open a new `0.5.3` item.

**Comment by ejgallego** [comment]
@Ptival this should fix the build:
```diff
@@ -197,7 +197,7 @@ let close_document () =
   let open Format in
   printf "Statistics:@\nSpecs:  %d@\nProofs: %d@\nMisc:   %d@\n%!" stats.specs stats.proofs stats.misc
 
-let sercomp debug printer async coq_path lp1 lp2 in_file omit_loc omit_att =
+let sercomp debug printer async coq_path ml_path lp1 lp2 in_file omit_loc omit_att =
 
   (* serlib initialization *)
   Serlib_init.init ~omit_loc ~omit_att;
@@ -209,7 +209,7 @@ let sercomp debug printer async coq_path lp1 lp2 in_file omit_loc omit_att =
   let in_pa   = Pcoq.Gram.parsable ~file:(Loc.InFile in_file) in_strm in
   let pp_sexp = Sertop_ser.select_printer printer        in
 
-  let iload_path = coq_loadpath_default ~implicit:true ~coq_path @ lp1 @ lp2 in
+  let iload_path = coq_loadpath_default ~implicit:true ~coq_path @ ml_path @ lp1 @ lp2 in
 
   (* let pp_opt  fb   = Sertop_util.feedback_opt_filter fb                in
    * let pp_feed fb   = Option.iter (fun fb -> pp_answer (SP.Feedback fb)) (pp_opt fb) in *)
```
please update and update the changelog. Thanks for your patiente!

**Comment by Ptival** [comment]
Done!

**Comment by ejgallego** [comment]
Thanks a lot! Will upload to OPAM ASAP.

**Pull request merged**